src/ops.rs

```diff
@@ -107,14 +107,14 @@ pub unsafe fn add_n_m(
     let pa = a.as_ptr() as *const limb_t;
     let pb = b.as_ptr() as *const limb_t;
     let pout = output.as_mut_ptr() as *mut limb_t;
-    let scratch_space = scratch_space.as_mut_ptr() as *mut limb_t;
+    let scratch_space_ = scratch_space.as_mut_ptr() as *mut limb_t;
```

Comment:
Just wanted to prevent shadowing to be able to print the original `scratch_space` variable in the end of this method to make sure that it does not reply work. Forgot to change it back